tools/clang/lib/SPIRV/SPIRVEmitter.cpp

```diff
@@ -2656,7 +2671,8 @@ uint32_t SPIRVEmitter::processGetDimensions(const CXXMemberCallExpr *expr) {
              TypeTranslator::isConsumeStructuredBuffer(objectType)) {
     return processByteAddressBufferStructuredBufferGetDimensions(expr);
   } else {
-    emitError("GetDimensions not implmented for the given type yet.");
+    emitError("GetDimensions not implmented for the given type yet.",
```

Comment:
GetDimensions() of the given object type unimplemented

Reply:
Done